daceml/onnx/onnx_importer.py

```diff
@@ -423,18 +431,17 @@ def compile_and_init(self) -> compiled_sdfg.CompiledSDFG:
         # copy all parameters to the device
         self.initialized_parameters = {}
         for name, arr in self.weights.items():
-            desc = self.sdfg.arrays[clean_onnx_name(name)]
-            if type(desc) is dt.Scalar:
-                self.initialized_parameters[clean_onnx_name(
-                    name)] = arr.cpu().numpy()[()]
-            else:
-                cuda = is_cuda(desc.storage)
-                self.initialized_parameters[clean_onnx_name(
-                    name)] = arr.cuda() if cuda else arr
+            if clean_onnx_name(name) in self.sdfg.arrays:
```

Comment:
What happens if it isn't?

Reply:
All weights are added to the SDFG on import. If it's missing here, it means a transformation removed it